http/src/test/scala/org/http4s/blaze/http/endtoend/scaffolds/AsyncHttpClientScaffold.scala

```diff
@@ -0,0 +1,68 @@
+package org.http4s.blaze.http.endtoend.scaffolds
+
+import java.util.concurrent.TimeUnit
+
+import org.asynchttpclient.{Dsl, Request, RequestBuilder, Response}
+import org.http4s.blaze.http.{HttpRequest, HttpResponsePrelude}
+
+import scala.collection.JavaConverters._
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+/** Basic scaffold for running http requests against a server
+ *
+  * @param timeout timeout for request. Must be finite and greater than 0
+  */
+private[endtoend] final class AsyncHttpClientScaffold(timeout: Duration) extends ClientScaffold(1, 1) {
```

Comment:
Why AsyncHttpClient instead of Http1ClientScaffold in line with the other naming conventions in the scaffolds?

Reply:
The name does suck, though I'd like to keep the name broadcasting that it's the `async-http-client` since I'd like to have a couple implementations at the end of the day.